src/Microsoft.DocAsCode.MarkdigEngine.Extensions/Noloc/NolocParser.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Microsoft.DocAsCode.MarkdigEngine.Extensions
+{
+    using Markdig.Helpers;
+    using Markdig.Parsers;
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+
+    public class NolocParser : InlineParser
+    {
+        // syntax => ::: noloc text="{content}" :::
+        private const string StartString = "::: noloc text=\"";
+        private const string EndString = "\" :::";
+
+        public NolocParser()
+        {
+            OpeningCharacters = new[] { ':' };
+        }
+
+        public override bool Match(InlineProcessor processor, ref StringSlice slice)
+        {
+            if (!ExtensionsHelper.MatchStart(ref slice, StartString, false))
```

Comment:
does the spec said it needs to be exactly match `::: noloc text="`? should `:::no-lock     text  = "` work?

Reply:
This is not specified in spec, I just prefer to do it strictly. I have called it out and let`s wait for PM`s confirmation.

Reply:
The current design is no optional spaces allowed.